ghu_web/ghu_main/static/ghu_main/css/base.css

```diff
@@ -54,3 +58,32 @@
     background: #4aaec6; /* Fallback */
     background: linear-gradient(#4aaec6, #2dc5ea);
 }
+
+/* Mobile Size */
+@media (max-width: 770px ) {
+    #logo {
+        margin-top: 18px;
+        margin-bottom: 18px;
+    }
+
+    #logo a {
+        font-size: 5.2vw;
+    }
+
+    #jumbotron-title {
+        margin-top: 210px;
+    }
+
+    #jumbotron-title p {
+        margin-left: 10px;
+        padding-left: 30px;
+        font-size: 3.7vw;
+    }
+
+    #feedback {
+      width: 70px;
+      height: 35px;
+      line-height: 29px;
+      font-size: 3.3vw;
```

Comment:
I'm thinking we don't touch `#feedback` in this block. `vw` means 1% of the viewport width, the `font-size` directive here can generate some undesired results (look at the feedback button):

![screen shot 2017-09-18 at 22 41 30](https://user-images.githubusercontent.com/431756/30573587-c019b7bc-9cc2-11e7-83bc-f08c0f970e45.png)

Reply:
Instead, if we're worried about it overlapping the logo, we could instead increase the top margin of the logo in this `@media` block

Reply:
Yea I agree